pkg/operator/ceph/config/keyring/admin.go

```diff
@@ -30,8 +30,8 @@ const (
 	adminKeyringResourceName          = "rook-ceph-admin"
 	crashCollectorKeyringResourceName = "rook-ceph-crash-collector"
 
-	adminKeyringTemplate = `
-[client.admin]
+	operatorAdminKeyringTemplate = `
+[client.rookoperator]
```

Comment:
This might be the issue... we need to match the expected username, so this needs to be client.admin initially.